src/app/view/IAP/hooks/useClouderyOffer.ts

```diff
@@ -12,15 +14,28 @@ import {
 
 const log = Minilog('💳 Cloudery Offer')
 
+const IOS_OFFERS = ['test_comfort_monthly', 'test_super_comfort_monthly']
+const ANDROID_OFFERS = ['test_comfort', 'test_super_comfort']
```

Comment:
Is it mandatory to store the offers within the flagship app? (also `test_` seems weird)

Reply:
Yes this is mandatory. The app choose which offer to handle or not. I think this has been chosen by Android and iOS stores in order to allow creating new offers that targets only most recent app versions. The app won't display a new offer until we explicitly declare it.

However we can put it in the stack, but this would re-introduce the problem I thing the stores tried to fix.

Reply:
I forgot to answer the `test_` part. This is just the offers we use for tests. For now the feature is behind a flag so this shouldn't be problematic.

But that being said, maybe we should avoid to publish a version with those values so we won't have to check the version when adding the flag to everyone.

I'll check with @anthonycozy if we can create final values here

Reply:
I put final offers here: https://github.com/cozy/cozy-flagship-app/pull/991/commits/b4b31984bdfaa5a0fa14b8720afc2b5d72a9b756#diff-de03f5378e032b3c91fdfaf88f0ad11e3569fe199c7d3db80d5f235c0fe1b904R17-R21